src/test/resources/org/prebid/server/it/openrtb2/sharethrough/test-auction-sharethrough-response.json

```diff
@@ -0,0 +1,183 @@
+          "dealid": "dealId",
+          "ext": {
+            "prebid": {
+              "cache": {
+                "bids": {
+                  "cacheId": "a03927f1-86e1-48fb-9efc-da9104a44e39",
+                  "url": "http://localhost:8090/cache?uuid=a03927f1-86e1-48fb-9efc-da9104a44e39"
+                }
+              },
+              "targeting": {
+                "hb_bidder": "sharethrough",
+                "hb_bidder_sharethrough": "sharethrough",
+                "hb_cache_host": "localhost:8090",
+                "hb_cache_host_sharethrough": "localhost:8090",
+                "hb_cache_id": "a03927f1-86e1-48fb-9efc-da9104a44e39",
+                "hb_cache_id_sharethrough": "a03927f1-86e1-48fb-9efc-da9104a44e39",
+                "hb_cache_path": "/cache",
+                "hb_cache_path_sharethrough": "/cache",
+                "hb_deal": "dealId",
+                "hb_deal_sharethrough": "dealId",
+                "hb_pb": "10.00",
+                "hb_pb_sharethrough": "10.00",
+                "hb_size": "50x50",
+                "hb_size_sharethrough": "50x50"
+              },
+              "type": "native"
+            }
+          },
+          "h": 50,
+          "id": "bid",
+          "impid": "bid",
+          "price": 10,
+          "w": 50
+        }
+      ],
+      "group": 0,
+      "seat": "sharethrough"
+    }
+  ],
+  "cur": "USD",
+  "ext": {
+    "debug": {
+      "httpcalls": {
+        "cache": [
+          {
+            "requestbody": "{\"puts\":[{\"type\":\"json\",\"value\":{\"id\":\"bid\",\"impid\":\"bid\",\"price\":10,\"adm\":\"<img src=\\\"//b.sharethrough.com/butler?type=s2s-win&arid=arid\\\" />\\n\\t\\t<div data-str-native-key=\\\"abc123\\\" data-stx-response-name=\\\"str_response_bid\\\"></div>\\n\\t\\t<script>var str_response_bid = \\\"eyJhZHNlcnZlclJlcXVlc3RJZCI6ImFyaWQiLCJiaWRJZCI6ImJpZCIsImNyZWF0aXZlcyI6W3siY3BtIjoxMCwiY3JlYXRpdmUiOnsiY2FtcGFpZ25fa2V5IjoiY21wS2V5IiwiY3JlYXRpdmVfa2V5IjoiY3JlYUtleSIsImRlYWxfaWQiOiJkZWFsSWQifSwidmVyc2lvbiI6MH1dfQ==\\\"</script>\\n\\t\\t\\t<script src=\\\"//native.sharethrough.com/assets/sfp.js\\\"></script>\\n\",\"adid\":\"arid\",\"cid\":\"cmpKey\",\"crid\":\"creaKey\",\"dealid\":\"dealId\",\"w\":50,\"h\":50}}]}",
+            "responsebody": "{\"responses\":[{\"uuid\":\"a03927f1-86e1-48fb-9efc-da9104a44e39\"}]}",
+            "status": 200,
+            "uri": "http://localhost:8090/cache"
+          }
+        ],
+        "sharethrough": [
+          {
+            "responsebody": "{\"adserverRequestId\":\"arid\",\"bidId\":\"bid\",\"creatives\":[{\"cpm\":10,\"creative\":{\"campaign_key\":\"cmpKey\",\"creative_key\":\"creaKey\",\"deal_id\":\"dealId\"},\"version\":0}]}",
+            "status": 200,
+            "uri": "http://localhost:8090/sharethrough-exchange?placement_key=abc123&bidId=bid&consent_required=false&consent_string=consentValue&instant_play_capable=true&stayInIframe=true&height=50&width=50&supplyId=FGMrCMMc&strVersion=1.0.0"
```

Comment:
`http://localhost:8090/sharethrough-exchange` replace with the `{{ vrtcal.exchange_uri }}` placeholder.